### DISCUSSION TITLE
Range Input Accent Color with a Single Hex Value Not Switching Correctly Between Light and Dark Modes in Tailwind CSS

### DISCUSSION THREAD

**Discussion by MonaAghili** [tailwindlabs/tailwindcss#15300] [Help] [answered]
version of Tailwind CSS v3.4.3
We are using chrome and brave
We use Windows and MacOS
We are using `Phoenix LiveView` with `esbuild` for JavaScript bundling and asset management.

**Reproduction URL**  
[https://play.tailwindcss.com/DF91Zxjl2M](https://play.tailwindcss.com/DF91Zxjl2M)

**The issue**  
I have an `input` range field with a default `accent` color specified for both light and dark modes using the `accent-[color]` class. However, in light mode, the bar of the range field stays dark instead of using the specified light mode color. I tested this in the Tailwind Play playground, and the issue persists there as well, suggesting it might not be project-specific.

Here’s an example of the code I used:
```html
<input type="range" value="20" name="form-example-3" class="w-full accent-[#CA8D01] dark:accent-[#FDC034]" max="100" min="0" />
<input type="range" value="40" name="form-example-2" class="w-full accent-[#CA8D01]" max="100" min="0" />
```

I expect the range bar to change its accent color correctly between light and dark modes, but it remains dark in light mode. This issue also occurs in the Tailwind Play environment, as seen in the provided link.

Any guidance on how to resolve this would be much appreciated!

**Comment by adamwathan** [comment]
Hmm the color changes in dark mode for me in your demo:

https://github.com/user-attachments/assets/254a9cf3-807d-4d49-8b9e-9eff2fdace1c

Are you seeing different behavior?

**Reply by MonaAghili** [reply]
In your video, the main bar of the range remains blackish.

As you can see in the video below, of my website, when I switch between light and dark mode, the bar color changes for most of the range fields. However, the yellow one still has this issue.

Additionally, in the playground example
https://play.tailwindcss.com/Q5Pb8PRu2V, the bar color doesn’t change at all, even when I provide more colors!

https://github.com/user-attachments/assets/4819e8ac-3503-4016-a4d0-5edaba9a7202

**Reply by adamwathan** [reply]
This is just how the browser behaves when you set a light accent color like yellow — it makes the bar dark to preserve sufficient contrast between each side of the range so it's easy to see. I don't think it's possible to change this with CSS as far as I know, the only thing you can change is the colored part, not the empty part.

**Reply by adamwathan** [reply, marked as answer]
Can see here how the bar color is affected by the lightness of the chosen accent color:

https://github.com/user-attachments/assets/9cef5494-9dc1-4612-a42d-f00b30519cc6
